.github/ISSUE_TEMPLATE/help.md

```diff
@@ -5,42 +5,71 @@ labels: 'type: question'
 
 ---
 
-<!-- EssentialsX help guide
+<!-- EssentialsX help request guide
 
-Don't write inside the arrows as they will be hidden when you post your issue.
+NOTE: Failure to fill out this template properly may result in your issue being
+      delayed or ignored.
 
-Want faster support? Come join our Discord server to get help sooner:
-https://discord.gg/F7gexAQ
+Don't put anything inside this block, as it will be hidden when you post your
+issue. Make sure to also not type between any arrows in the template, as this
+text will also be hidden.
 
-If you're happy to wait (or you were sent here from Discord), read on:
+Want faster support? Come join our Discord server to get help from both
+developers and community members: https://discord.gg/F7gexAQ
 
-1.  Fill out the template.
-      This will help us understand what problem you've encountered and help us
-      find a solution.
+If you're happy to wait (or you were sent here from Discord), read on:
 
-2.  Check the Common Issues page.
+1.  Check the Common Issues page.
       Read through the wiki page to see if you've encountered a regular issue:
       https://essentialsx.net/wiki/Common-Issues.html
 
-3.  Delete this line and all above lines before posting your issue!       -->
+2.  Fill out the template.
+      Run the commands in the console. Don't just put "latest" as a version, or
+      we will ignore it. This will help us understand what problem you've
+      encountered and help us find a solution.
+
+3.  When linking logs or config files, do not attach them to the post!
+      Copy and paste any logs into https://gist.github.com/, then paste a
+      link to them in the relevant parts of the template. Do not use Hastebin
+      or Pastebin, as this can cause issues when trying to solve issues in the
+      future.
+      DO NOT drag logs into this box!
+
+4.  If you are encountering a performance issue, please include a link to a
+    Timings and/or profiler report, both before and after your PR.
+
+-->
 
 ### Information
 
-Full output of `/ess version`:
+**Full output of `/ess version`:**
+<!--
+    Paste the output of `/ess version` inside the ```s, preferably copied
+    directly from the console.
+-->
+```
 
-<!-- Replace this with the command output -->
+```
 
-**Server log:** <!-- Upload `logs/latest.log` to https://gist.github.com/ -->
+**Server log:**
+<!--
+    Upload `logs/latest.log` to https://gist.github.com/ then paste the link
+    below this block.
```

Comment:
This section of the template has always been under hot debate by people worried about privacy. Perhaps good to give them some peace of mind; this shouldn't affect us too much and we can always remind them to re-submit if they fail to include critical information.
```suggestion
    Upload `logs/latest.log` to https://gist.github.com/ then paste the link
    below this block. You may remove any sensitive information, but please
    make it clear what was removed.
```